packages/spec/src/index.ts

```diff
@@ -26,24 +38,22 @@ export declare namespace v1 {
      */
     readonly validate: (
       value: unknown,
-    ) => StandardResult<Output> | Promise<StandardResult<Output>>;
+    ) => Result<Output> | Promise<Result<Output>>;
     /**
      * Inferred types associated with the schema.
      */
-    readonly types?: StandardTypes<Input, Output> | undefined;
+    readonly types?: Declaration<Input, Output> | undefined;
```

Comment:
@colinhacks do you have an opinion on the name change?

Reply:
No strong opinions, this interface will virtually never be referenced by name. Could probably be inlined, but it's also fine as is

Reply:
I think I prefer to keep the current name. `Type` seems to me to be more common than `Declaration`, and I also prefer to call the key `types`, but... this is not a very strong opinion. At the moment, the most important thing is to agree on an implementation that is v1 RC ready, and to help integrate the spec into other libraries. I hope to find more time for this around or after Christmas.